commit 1547a6ed0cda7df83729069e8589609c6eb332f7
Author: Randy Fay <randy@randyfay.com>
Date:   Sun Jun 9 15:21:23 2013 -0600

    Minor patch to services to make HEAD work like GET

diff --git a/servers/rest_server/includes/RESTServer.inc b/servers/rest_server/includes/RESTServer.inc
index 38e9a7c..8b1ba73 100755
--- a/servers/rest_server/includes/RESTServer.inc
+++ b/servers/rest_server/includes/RESTServer.inc
@@ -537,7 +537,7 @@ class RESTServer {
     $operation = NULL;
 
     // Use the index handler for all empty path GET-requests
-    if (!$pc && $method == 'GET') {
+    if (!$pc && ($method == 'GET' || $method == 'HEAD')) {
       $class = 'operations';
       $operation = 'index';
     }
